frontend/js/src/donors/Donors.tsx

```diff
@@ -98,13 +98,22 @@ function Donors() {
                 {donor.musicbrainz_id &&
                   (donor.is_listenbrainz_user ? (
                     <Link
-                      to={`/user/${donor.musicbrainz_id}`}
+                      to={`${
+                        !donor.is_listenbrainz_user && "https://musicbrainz.org"
```

Comment:
I suspect this condition is never truthy,aas you are already inside a conditional check of  `donor.is_listenbrainz_user ? (...`